src/main/java/com/faforever/client/chat/PrivateChatTabController.java

```diff
@@ -83,13 +90,20 @@ public void setReceiver(String username) {
     super.setReceiver(username);
     privateChatTabRoot.setId(username);
     privateChatTabRoot.setText(username);
+    playerService.getPlayerByNameIfOnline(username)
+        .map(PlayerBean::getAvatar)
+        .map(avatarService::loadAvatar).ifPresent(image -> JavaFxUtil.runLater(() -> {
+          avatarImageView.setVisible(true);
+          avatarImageView.setImage(image);
+        }));
```

Comment:
A listener will also need to be added if the player changes their avatar if the tab is open